Update usecontractread.mdx (#1111)
thirdweb-dev/docs: docs/react/hooks/core/usecontractread.mdx

If you provide an additional argument to the hook, it will be used as the `CallOverrides` object send with your request.

> To retrieve the sender's address using msg.sender in view functions within your smart contract, it is essential to include the property {from: 0X123} in order to access the relevant address information.

Reviewer: ```suggestion
To include the sender's address (msg.sender) when calling view functions within your smart contract, include the property {from: 0X123} passing the relevant address.
```
